dev/io.openliberty.webcontainer.servlet.6.1.internal.factories/bnd.bnd

```diff
@@ -0,0 +1,108 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 17
+javac.target: 17
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
+
+Bundle-Name: WAS WebContainer
+Bundle-SymbolicName: io.openliberty.webcontainer.servlet.6.1.internal.factories
+Bundle-Description: WAS WebContainer with object factories for Servlet 6.1 support
+WLP-ServerName: IBM WebSphere Application Server
+WLP-ServerVersion: ${liberty.beta.version}
+
+-dsannotations: \
+  io.openliberty.webcontainer.servlet.internal.async.factory.AsyncContextFactory61Impl, \
+  io.openliberty.webcontainer.servlet.internal.osgi.container.config.factory.WebAppConfiguratorFactory61Impl, \
+  io.openliberty.webcontainer.servlet.internal.osgi.response.factory.IRequestResponseFactory61Impl, \
+  io.openliberty.webcontainer.servlet.internal.osgi.srt.factory.SRTConnectionContextPool61Impl, \
+  io.openliberty.webcontainer.servlet.internal.osgi.webapp.factory.WebAppFactory61Impl, \
+  io.openliberty.webcontainer.servlet.internal.servlet.factory.CacheServletWrapperFactory61Impl, \
+  io.openliberty.webcontainer.servlet.internal.session.impl.factory.SessionContextRegistryImplFactory61Impl, \
+  io.openliberty.webcontainer.servlet.internal.util.factory.URIMatcherFactory61Impl
+
+#Need to update to v61 ??? 
```

Comment:
Where is this used today? I know we use the properties file within the WC components. Can you look to see where this is used to determine if we need an update?

Reply:
updated